libraries/timeline-view/src/main/java/com/trendyol/timelineview/TimelineItemViewState.kt

```diff
@@ -0,0 +1,34 @@
+package com.trendyol.timelineview
+
+import android.graphics.Color
+import android.view.View
+
+class TimelineItemViewState(
+    private val timelineItem: TimelineItem,
+    val dotSize: Float,
+    val borderWidth: Float,
+    val textSize: Float
+) {
+
+    fun getOutsideColor(): Int = Color.parseColor(timelineItem.outsideColor)
+
+    fun getInsideColor(): Int = Color.parseColor(timelineItem.insideColor)
+
+    fun getText(): String = timelineItem.text
+
+    fun getTextColor(): Int = Color.parseColor(timelineItem.textColor)
+
+    fun getLeftLineColor(): Int = Color.parseColor(timelineItem.leftLineColor ?: "#000000")
```

Comment:
Can you move constants into `companion object` block as private values?

Reply:
done